sormas-ui/src/main/java/de/symeda/sormas/ui/utils/components/MultiSelectFiles.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * ******************************************************************************
+ * * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * * Copyright © 2016-2024 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ * *
+ * * This program is free software: you can redistribute it and/or modify
+ * * it under the terms of the GNU General Public License as published by
+ * * the Free Software Foundation, either version 3 of the License, or
+ * * (at your option) any later version.
+ * *
+ * * This program is distributed in the hope that it will be useful,
+ * * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * * GNU General Public License for more details.
+ * *
+ * * You should have received a copy of the GNU General Public License
+ * * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ * ******************************************************************************
+ */
+
+package de.symeda.sormas.ui.utils.components;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import com.vaadin.icons.VaadinIcons;
+import com.vaadin.shared.ui.MarginInfo;
+import com.vaadin.ui.Button;
+import com.vaadin.ui.Component;
+import com.vaadin.ui.HorizontalLayout;
+import com.vaadin.ui.Label;
+import com.vaadin.ui.VerticalLayout;
+import com.vaadin.ui.themes.ValoTheme;
+import com.vaadin.v7.data.util.converter.Converter;
+import com.vaadin.v7.ui.CustomField;
+
+import de.symeda.sormas.ui.utils.ButtonHelper;
+import de.symeda.sormas.ui.utils.VaadinUiUtil;
+
+public class MultiSelectFiles<T> extends CustomField<Set<T>> {
+
+	private VerticalLayout labelLayout = new VerticalLayout();
+	private Map<String, T> selectedItemsWithCaption = new HashMap<>();
+
+	private Runnable deleteAttachmentCallBack;
```

Comment:
this field is not used, please delete it

Reply:
removed it